lib/mathlib.cpp

```diff
@@ -143,6 +143,8 @@ double MathLib::toDoubleNumber(const std::string &str)
     // nullcheck
     else if (isNullValue(str))
         return 0.0;
+    else if (isFloat(str)) // Workaround libc++ bug at http://llvm.org/bugs/show_bug.cgi?id=17782
```

Comment:
This is explained in PR 228 and 562 as follows (the situation is Mac specific)
* If one uses clang + libc++ (default), we get a lot of test suite failures due to http://llvm.org/bugs/show_bug.cgi?id=17782
* To have the test suite succeed, one needs to use GNU's libstdc++ instead of libc++... but it does not provide cbegin() / cend(), and cppcheck does not build